packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -333,6 +335,8 @@ export const prepareCosmosOrchestrationAccountKit = (
           return account;
         },
         getUpdater() {
+          if (!this.state.topicKit)
+            throw Fail`No topicKit; storageNode not provided`;
```

Comment:
consider this, which I think suffices:
```suggestion
            throw Fail`no topicKit`;
```